tests/deploy/e2e/test_dryrun.py

```diff
@@ -243,6 +243,7 @@ def assert_handler_failed(resource, msg):
     response = await client.get_resource(environment, "test::DoesNotExist[agent1,key=key2],v=%d" % version, logs=True)
     assert response.code == 200
 
+    # TODO: Do we want this behaviour?
```

Comment:
Was this behaviour intentional?

i.e. We log an `unavailable` resource action as well as a `dryrun` action. And we log them in such a way that the unavailable is the first to be returned when we fetch the logs?

Reply:
I don't think it is very valuable to have two log lines for the same thing

Reply:
One of them has the status as `dry` and the other as `unavailable`. It is the behaviour that was present before my changes.

Should we keep only the `dry` or the `unavailable`?

Reply:
I agree, which do you want to keep?

I'm inclined to keep the `unavailable` one. I think that makes the most sense